lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -89,7 +89,7 @@ class GradingPolicySerializer(serializers.Serializer):
     def get_assignment_policies(self, grading_policy):
         return [{
             'num_droppable': assignment_policy['drop_count'],
-            'num_total': assignment_policy['min_count'],
+            'num_total': float(assignment_policy['min_count']),
```

Comment:
For some reason only this value is coming up as a string, which is breaking things in the MFE. Not sure if it's worth being defensive and casting the rest of these values?

Reply:
Also maybe worth fixing on backend? Feels like it should be a float in json?

Is drop_count not a problem? Or weight?

Reply:
Sorry, I thought I was in MFE - so this is backend.